docs/technical_reference/unit_models/electroNP_ZO.rst

```diff
@@ -0,0 +1,83 @@
+ElectroNP (ZO)
+====================
+This electrochemical nutrient removal (ElectroNP) unit model
+   * is a zero-order model that enables the user to specify performance in terms of solute rejection
+   * supports a single liquid phase only
+   * supports steady-state only
+   * assumes isothermal conditions
+
+.. index::
+   pair: watertap.unit_models.electroNP_ZO;electroNP_ZO
+
+.. currentmodule:: watertap.unit_models.electroNP_ZO
+
+Degrees of Freedom
+------------------
+The zero-order electroNP model has at least 6 degrees of freedom that should be fixed for the unit to be fully specified.
+Typically, the following variables are fixed:
+
+   * inlet volumetric flow rate
+   * inlet temperature
+   * inlet pressure
+   * electricity intensity
+   * dosage of magnesium chloride per phosphorus removal
+
+There are 1 degree of freedom for each solute in a given property model:
+
+   * inlet solute concentration
+
+Model Structure
+------------------
+This electroNP model is built based on IDAES Separator module. It consists of one inlet port and two outlet ports as
+treated and byproducts. The ``SplittingType`` is set to ``componentFlow``.
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Time", ":math:`t`", "[0]"
+   "Phases", ":math:`p`", "['Liq']"
+   "Components", ":math:`j`", "['H2O', 'Solute']*"
+
+\*Solute depends on the imported property model, and more than one solute can be provided.
+
+Variables
+----------
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Mass recovery fraction of water", ":math:`R_{H2O}`", "recovery_frac_mass_H2O", "[t]", ":math:`\text{dimensionless}`"
+   "Removal fraction of component", ":math:`r_j`", "removal_frac_mass_comp", "[t, 'byproduct', j]", ":math:`\text{dimensionless}`"
+   "Electricity consumption", ":math:`E`", "electricity", "[t]", ":math:`\text{kW}`"
+   "Electricity intensity with respect to phosphorus removal", ":math:`EI`", "energy_electric_flow_mass", " ", ":math:`\text{kWh/kg}`"
```

Comment:
```suggestion
   "Electricity intensity with respect to phosphorus removal", ":math:`EI`", "energy_electric_flow_mass", "None", ":math:`\text{kWh/kg}`"
```
I think the convention is to type "None" rather than leave a blank space